pom.xml

```diff
@@ -262,9 +262,11 @@
     <version.org.sonatype.plexus.plexus-sec-dispatcher>2.0</version.org.sonatype.plexus.plexus-sec-dispatcher>
     <version.org.sonatype.sisu>2.3.0</version.org.sonatype.sisu>
     <version.org.sonatype.sisu.sisu-guice>3.2.3</version.org.sonatype.sisu.sisu-guice>
-    <version.org.springframework>5.3.27</version.org.springframework>
-    <version.org.springframework.boot>2.5.15</version.org.springframework.boot>
+    <version.org.springframework>5.3.34</version.org.springframework>
+    <version.org.springframework.boot>2.7.18</version.org.springframework.boot>
     <version.org.springframework.osgi>1.2.1</version.org.springframework.osgi>
+    <spring-security.version>5.7.12</spring-security.version>
+    <tomcat.version>9.0.86</tomcat.version>
```

Comment:
Let's add a comment mentioning that these properties are overriding the version from `spring-boot-dependencies` BOM.